ffmpeg.yaml

```diff
@@ -155,6 +155,18 @@ subpackages:
       - uses: split/manpages
     description: ffmpeg manpages
 
+  - name: ffmpeg-qt-faststart
+    description: "Utility for optimizing MP4 files for HTTP streaming"
+    pipeline:
+      - runs: |
+          cd tools
+          # Use make to build qt-faststart specifically
+          make qt-faststart
+          # Create installation directory
+          mkdir -p "${{targets.contextdir}}"/usr/bin
+          # Install the binary
+          install -Dm755 qt-faststart "${{targets.contextdir}}"/usr/bin/
```

Comment:
would be great to have simple `qt-faststart --help` test

Reply:
I did better o7